lib/api/CAnomalyJob.cc

```diff
@@ -688,6 +688,13 @@ void CAnomalyJob::outputResults(core_t::TTime bucketStartTime) {
     this->writeOutAnnotations(annotations);
     this->writeOutResults(false, results, bucketStartTime, processingTime);
 
+    if (m_ModelConfig.modelPruneWindow() > core_t::TTime{0}) {
```

Comment:
```suggestion
    if (m_ModelConfig.modelPruneWindow() > 0) {
```

It seems unnecessarily verbose to case 0 in this line, and it shouldn't make any difference.